src/pybind/mgr/telemetry/module.py

```diff
@@ -939,6 +1034,140 @@ class EndPoint(enum.Enum):
         ceph = 'ceph'
         device = 'device'
 
+    def collection_delta(self, channels: Optional[List[str]] = None) -> Optional[List[Collection]]:
+        '''
+        Find collections that are available in the module, but are not in the db
+        '''
+        if self.db_collection is None:
+            return None
+
+        if not channels:
+            channels = ALL_CHANNELS
+        else:
+            for ch in channels:
+                if ch not in ALL_CHANNELS:
+                    self.log.debug(f"invalid channel name: {ch}")
+                    return None
+
+        new_collection : List[Collection] = []
+
+        for c in MODULE_COLLECTION:
```

Comment:
I would rename `c` to something more meaningful, like `collection` or `mod_collect` (module collection).